process/smartContract/process.go

```diff
@@ -219,6 +216,10 @@ func (sc *scProcessor) ExecuteSmartContractTransaction(
 		return nil
 	}
 
+	if check.IfNil(acntDst) {
```

Comment:
hmmm, interesting: you pushed this condition up until the very last moment, maybe, maybe it is a builtin function that can be executed

Reply:
yes. builtin functions are executed with other format.